components/widgets/inputs/users_emails_input.scss

```diff
@@ -203,3 +225,23 @@
         margin-left: 8px;
     }
 }
+
+.InputErrorBox {
+    margin-top: 2px;
+    width: 100%;
+    height: 60px;
+    background: rgba(197, 67, 72, 0.1);
```

Comment:
Does this correspond to a specific colour or should this be themed? Usually most colours correspond to a theme variable we have.

Reply:
This is a specific colour for the background of this error box. I created a variable for the colour so it can be used/overridden if needed.